funnel/views/schedule.py

```diff
@@ -247,8 +247,13 @@ def schedule_edit(profile, space):
             } for proposal in space.proposals_all if proposal.confirmed and not proposal.session],
         'scheduled': session_data(space.sessions, timezone=space.timezone, with_modal_url='edit', with_delete_url=True)
         }
+    # Set the proper range for the calendar to allow for date changes
+    first_session_date = Session.query.filter_by(proposal_space=space).order_by("created_at asc").first().start
+    last_session_date = Session.query.filter_by(proposal_space=space).order_by("created_at desc").first().start
+    from_date = first_session_date if first_session_date.date() < space.date else space.date
+    to_date = last_session_date if last_session_date.date() > space.date_upto else space.date_upto
```

Comment:
Interesting use case; hasn't been covered here. One option is to provide a 'bulk edit' option that remaps the dates of the sessions to the new dates. Also, dragging multiple sessions across weeks or months can be cumbersome.